src/SamplesApp/SamplesApp.UITests/SamplesApp.UITests.csproj

```diff
@@ -11,6 +11,8 @@
 		<NoWarn>$(NoWarn);SYSLIB1045;CS1998</NoWarn>
 	</PropertyGroup>
 
+	<Import Project="../../targetframework-override-noplatform.props" />
```

Comment:
I think it can be kept as NetPrevious. CI may complain that the .NET 8 runtime isn't installed, in this case we can just add a step that installs the runtime.